tests/cpu-limit-unset/00-errors.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test
+spec:
+  template:
+    spec:
+      containers:
+        - resources:
+            limits:
+              cpu: 10m
```

Comment:
How does this file assert that the test should fail?

Reply:
From the [kuttl documentation](https://kuttl.dev/docs/testing/asserts-errors.html#format) they state that anything present in the `index-errors.yaml` file will cause a test error. As the CR removes the CPU limit even if it is present in the CRD spec, it should never be present.

We tested with the other resources, and those cause the test to fail, so this was the simplest way to check if a field was *not* present.